src/MensaWidget/MWFrameTest.class.st

```diff
@@ -38,18 +38,11 @@ MWFrameTest >> tearDown [
 
 {
 	#category : #tests,
-	#'squeak_changestamp' : 'tmc 6/6/2024 22:21'
+	#'squeak_changestamp' : 'Louis Christ 6/18/2024 17:30'
 }
 MWFrameTest >> testCantineCount [
 
-	self widgetFrame submorphsDo: [ :each | 
-		each class = ScrollPane ifTrue: 
-			[ each submorphsDo: [ :each2 |
-				each2 class = TransformationMorph ifTrue: 
-					[ each2 submorphsDo: [ :each3 |
-						each3 class = AlignmentMorph ifTrue: [
-							self assert: each3 subclasses size = 
-								MWDataGettingWrapper canteenDict size]]]]]]
+	self assert: self widgetFrame submorphs first contentMorph submorphs size = MWDataGettingWrapper canteenDict size
```

Comment:
i think for better understanding of the testcase, something like widgetframe submorphs first could be put into a variable with descriptive name